tests/test_budgetbytes.py

```diff
@@ -25,7 +25,9 @@ def test_total_time(self):
         self.assertEqual(45, self.harvester_class.total_time())
 
     def test_yields(self):
-        self.assertEqual("4 serving(s)", self.harvester_class.yields())
+        self.assertEqual(
+            "4 1.25 cups curry lentils + 1 cup rice", self.harvester_class.yields()
```

Comment:
Ref: #278 - this could cause problems for some clients, but I think is probably fine for now?  It'd be nice to find a consistent way to handle yield results.